health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/ChooseActivity.js

```
@@ -0,0 +1,84 @@
+        const userRoles = userInfo?.info?.roles?.map((roleData) => roleData?.code);
+      
+        const enabled = userRoles?.filter((role) => validRoles?.includes(role));
+      
+        return !(validRoles?.length > 0 && enabled?.length > 0);
+      };      
+
+    const activityCardData = [
+        { 
+            name: t("VALIDATE_N_APPROVE_POPULATION_DATA"), 
+            link: "", 
+            icon: <SVG.Population></SVG.Population>,
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)